Add info about how to disable session replay for older sdks (#23057)
DataDog/documentation: content/en/real_user_monitoring/session_replay/browser/_index.md

To stop session recordings, set `sessionReplaySampleRate` to `0`. This stops collecting data for the [Browser RUM & Session Replay plan][6].

> <div class="alert alert-warning">When using a version of the RUM Browser SDK older than v5.0.0, set `replaySampleRate` to `0`.</div>

Reviewer: ```suggestion
<div class="alert alert-warning">If you're using a version of the RUM Browser SDK previous to v5.0.0, set `replaySampleRate` to `0`.</div>
```
